controlButtons.py

```diff
@@ -12,30 +12,38 @@ def __init__(self):
         super().__init__()
         self.setGeometry(200, 200, 200, 200)
         self.buttonLayout()
-                
-    def buttonLayout(self):        
+
+
+
+    def buttonLayout(self):
         grid = QGridLayout()
         self.setLayout(grid)
-                
-        merge_left_button = QPushButton()
-        merge_left_button.resize( 100, 40)
-        icon = QtGui.QIcon(gui_config.ICONS["MERGE_LEFT"])
+
+        merge_left_button = QToolButton()
+        merge_left_button.setFixedSize(60, 60)
+        icon = QtGui.QIcon(gui_config.ICONS["UNDO"])
```

Comment:
As discussed, tool buttons look much better, good job setting up the dictionary for the file paths as well.